### ISSUE TITLE
Vite Build Error

### ISSUE THREAD

**Issue by seifmostafa73** [themesberg/flowbite-react#1115]
- [ ✅ ] I have searched the [Issues](https://github.com/themesberg/flowbite-react/issues) to see if this bug has already been reported
- [✅ ] I have tested the latest version

## Steps to reproduce

1. install tailwindCSS , flowbite, flowbite-react
2. build using vite ``` "scripts": {
    "start": "vite",
    "build": "tsc && vite build",
    "serve": "vite preview"
  },```

## Current behavior

Error occurs during build 
``` 

node_modules/flowbite-react/lib/esm/components/Flowbite/FlowbiteTheme.d.ts:1:44 - error TS2307: Cannot find module '~/src/components/FloatingLabel/FloatingLabel' or its corresponding type declarations.

1 import { FlowbiteFloatingLabelTheme } from '~/src/components/FloatingLabel/FloatingLabel';
                                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in node_modules/flowbite-react/lib/esm/components/Flowbite/FlowbiteTheme.d.ts:1
```
## Expected behavior

Build to finish without error
## Context

Note that
1. I can manually fix this error by simply going to the `FlowbiteTheme` and changing the import from ` import { FlowbiteFloatingLabelTheme } from '~/src/components/FloatingLabel/FloatingLabel';
` -> `import { FlowbiteFloatingLabelTheme } from '../FloatingLabel/FloatingLabel';`

2. this error only occurs on build

**Comment by seifmostafa73** [comment]
I believe this is also addressed here #1028
Is there a specific version that this occurred to downgrade to?

**Comment by seifmostafa73** [comment]
Rolling back to 0.6.2(from 0.6.4) didn't resolve the problem, however the 0.7.05 beta version resolved the problem
